api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -146,13 +147,28 @@ message TcpProxy {
   // The router selects an upstream cluster based on these weights.
   message WeightedCluster {
     message ClusterWeight {
-      // Name of the upstream cluster.
+      // Name of the upstream cluster. The cluster must exist in the
```

Comment:
I don't think this change is correct. The tcp_proxy handles the case of a missing cluster.

Reply:
Reverted in 82e4765.